engine/frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
could we use lowercase for the filename?

Reply:
i will keep it uppercase for now; App is usually uppercase as per react standards.